csvs/activities/ContentUserProgress.csv

```diff
@@ -0,0 +1,26 @@
+﻿ContentObjectId,UserId,CompletedDate,LastVisited,IsRead,NumRealVisits,NumFakeVisits,TotalTime,IsVisited,IsCurrentBookmark,IsSelfAssessComplete,LastModified,Version

```

Comment:
Are these supposed to be committed?

Reply:
Definitely not... must have been something while I rebased

Reply:
Hm I'm not sure where these came from... @CK-7vn are these familiar to you? If not, I can just delete them and push again

Reply:
Absolutely not...:'(

Reply:
Thanks a ton for the help with this Carolina! Sorry for the git mess up guys.